docs/src/tutorial/kubectl.md

```diff
@@ -0,0 +1,101 @@
+# Basic operations with Kubernetes using kubectl
+
+## What you will need
+
+Before you begin, make sure you have the following:
+
+- A bootstrapped Canonical K8s cluster (See [insert link]...)
+
+### 1. The Kubectl Command
+
+This commands interacts with the Kubernetes API server (kube-apiserver) and is the most commonly used command when working with Kubernetes, so let's take some time to familiarize ourselves with it.
+
+The `kubectl` command included with Canonical K8s is built from the original upstream source into the `k8s` snap you have installed.
+
+### 2. How To Use Kubectl
+
+You can access kubectl with the following command:
+
+```
+sudo k8s kubectl
```

Comment:
I would change this to `sudo k8s kubectl <command> ` or point the user to the help section `sudo k8s kubectl --help`